### PR TITLE
Add controlling access for pt-br

### PR DESCRIPTION
**Pull request by jralmaraz**
Submitting this PR to translate authorization and controlling-access pages to Brazilian Portuguese.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @rikatz

**Comment by divya-mohan0209** [comment]
Hello @rikatz @stormqueen1990! It has been ~2 months since this PR was submitted. Would it be possible for y'all to provide an initial review? I know y'all are busy, however is there any way we can support you so that this PR is reviewed?

**Comment by stormqueen1990** [comment]
Hi @divya-mohan0209!

Sure thing, I'll take a look in the next few days.

/cc

**Comment by stormqueen1990** [comment]
Hi @jralmaraz, thank you very much for your contribution! 🎉 

I noticed that you created a single PR with two pages. Could you please split the page `content/pt-br/docs/reference/access-authn-authz/authorization.md` into a separate PR?
It is our guideline to create separate PRs for each page and its related resources.

I will start reviewing the `controlling-access.md` page soon.

Thanks in advance!

**Comment by jralmaraz** [comment]
Hi @stormqueen1990 , thanks for picking this up and sorry for missing the guidelines, my bad.

I've split the files per PR as requested. Authorization translation PR was raised separately at:  https://github.com/kubernetes/website/pull/35854 

Let me know if there's any further action/fix, happy to address them.

Cheers.

**Comment by stormqueen1990** [comment]
> Hi @stormqueen1990 , thanks for picking this up and sorry for missing the guidelines, my bad.

No problem!

> I've split the files per PR as requested. Authorization translation PR was raised separately at: #35854

Thank you very much! I'll take a look at both PRs as soon as possible.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @jralmaraz, thank you very much for your contribution! 🎉 

I found some places that need changes. Please let me know if you need any clarification.

**Comment by jralmaraz** [comment]
> c818731

Hi @stormqueen1990 ! Thanks so much for the review and really sorry for the dummy mistakes, it seems like I left Brazil longer than I realize lol.

I have tried to address the gotchas with commit 
[c818731](https://github.com/kubernetes/website/pull/34097/commits/c818731ea677a1c069b37931e7967dfeb21c479e).

I'll wait for the preview deployment and go through the links and highlighted items later today or tomorrow.

Appreciate the help.

Cheers.

**Comment by jralmaraz** [comment]
I'm also keeping the authorization links here to the pt-br page, if that's ok. Let me know.

**Review by stormqueen1990 — CHANGES_REQUESTED** [review]
Hi @jralmaraz!

Found some small things that still need to be changed. I also noticed some leftover occurrences of words missing accents ("voce", "publica", "apos").

**Comment by jralmaraz** [comment]
Thanks a lot @stormqueen1990 . I'll go through the findings and hopefully push the fixes this week!

**Comment by jralmaraz** [comment]
I clicked by mistake on the re-request review item, my bad!!!

**Comment by stormqueen1990** [comment]
Hi @jralmaraz! It seems some items I had pointed out before have not been addressed, could you please take another look at those? I noticed there are still a few occurrences of words missing accents ("sao", "apos"). I think GitHub collapses the comments in the list, and that may be the reason they were left behind.

File: content/pt-br/docs/concepts/security/controlling-access.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+Os controladores de admissão não agem em requisições que apenas leem objetos.
+Quando vários controladores de admissão são configurados, eles são chamados em ordem.
+
+Isso é mostrado como etapa **3** no diagrama.
+
+Ao contrário dos módulos de autenticação e autorização, se algum módulo controlador de admissão
+rejeita, a solicitação é imediatamente rejeitada.
+
+Além de rejeitar objetos, os controladores de admissão também podem definir padrões complexos para
+campos.
```
[the remark this answers is not in the document]
Comment (jralmaraz) [reply content/pt-br/docs/concepts/security/controlling-access.md:121]: Thanks!

**Comment by jralmaraz** [comment]
> Hi @jralmaraz! It seems some items I had pointed out before have not been addressed, could you please take another look at those?
> 
> I noticed there are still some occurrences of words missing accents ("sao", "apos", among others). I think GitHub collapses the comments in the list, and that may be the reason they were left behind.

Indeed @stormqueen1990 , thanks again! I've pushed some changes, let me know if they look ok now.

File: content/pt-br/docs/concepts/security/controlling-access.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+<!-- overview -->
+Esta página fornece uma visão geral do controle de acesso à API do Kubernetes.
+
+<!-- body -->
+Usuários podem acessar a [API do Kubernetes](/pt-br/docs/concepts/overview/kubernetes-api/) utilizando `kubectl`,
+bibliotecas, ou executando requisições REST.  Ambos, usuários humanos e
+[Contas de serviço do Kubernetes](/pt-br/docs/tasks/configure-pod-container/configure-service-account/) podem ser autorizados
+a acessar a API.
+Quando uma requisição chega a API, ela passa por diversos estágios,
```
[the remark this answers is not in the document]
Comment (jralmaraz) [reply content/pt-br/docs/concepts/security/controlling-access.md]: fixed on b0051ac2e1

**Comment by edsoncelio** [comment]
Looks good to me, thank you @jralmaraz!
/lgtm

**Comment by stormqueen1990** [comment]
/approve

Thanks!

**Pull request merged**